lib/endpoints/workflow-instances.js

```diff
@@ -86,6 +87,9 @@ const endpoint = (client) => ({
 
     if (opts.documentDelivery) {
       form.append('document_delivery', 'true');
+      if (opts.documentDeliveryType) {
```

Comment:
Can we update the code to reflect the spacing as the rest of the file? Seems to have an extra indent for the lines added here (90-92)

Reply:
Line 90 should he a blank space and lines 91-93 should be where the new code snippet is places with the correct indentation.

Reply:
Definitely. I'll get to work on that now.

Reply:
@latoyazamill, you're right, this is a slight departure from how we check for options, but in this case I think @behr-davide is correct in embedding it inside the check for `opts.documentDelivery` as it is a dependency — there's no reason to check for the document delivery type if `documentDelivery` is falsey. It would be perfectly fine to move it outside of the conditional, but then we'd be doing more work than we'd need to

Reply:
Sounds good to me! Thanks for the breakdown 👍